handbook/business-operations/security-audits.md

```diff
@@ -2,7 +2,32 @@
 This page contains explanations of the latest external security audits performed on Fleet software.
 
 ## June 2023 penetration testing of Fleet 4.33 
-Available on request.
+In June 2023, [Latacora](https://www.latacora.com/) performed an application penetration assessment of the application from Fleet.
+
+An application penetration test captures a point-in-time assessment of vulnerabilities, misconfigurations, and gaps in applications that could allow an attacker to compromise the security, availability, processing integrity, confidentiality, and privacy (SAPCP) of sensitive data and application resources. An application penetration
+
+test simulates the capabilities of a real adversary, but accelerates testing by using information provided by the target company.
+
+You can find the full report here: [2023-06-09-fleet-penetration-test.pdf](https://github.com/fleetdm/fleet/raw/main/docs/files/2023-06-09-fleet-penetration-test.pdf).
+
+### Findings
+#### 1 - Stored Cross-Site Scripting (XSS) in Tooltip
+| Type                | Latacora Severity |
+| ------------------- | -------------- |
+| Cross-Site Scripting| High risk      |
+
+All tooltips using the tipContent tag are set using dangerouslySetInnerHTML. This allows manipulation of the DOM without sanitization. If a user can control the content sent to this function, it can lead to a cross-site scripting vulnerability. 
+
+- Resolved. Resolution information TBA
+
+#### 2 - Broken Authorization Leads to Observers Able to Add Hosts
+| Type                | Latacora Severity |
```

Comment:
Same

Reply:
@Sampfluger88 Our 2022 pen test explanation shares this same format - "Lares Severity". It might be odd to change the capitalization  on only the new one. Lmk what you think there.